reference/5.1/Microsoft.PowerShell.Core/About/about_PSModulePath.md

```diff
@@ -0,0 +1,181 @@
+
+The value of `$env:PSModulePath` is constructed each time PowerShell starts.
+The value varies by version of PowerShell and how it is launched.
+
+### Windows PowerShell startup
+
+Windows PowerShell uses the following logic to construct the `PSModulePath` at
+startup:
+
+- If `PSModulePath` doesn't exist, combine **CurrentUser**, **AllUsers**, and
+  the `$PSHOME` modules paths
+- If `PSModulePath` does exist:
+  - If `PSModulePath` contains `$PSHOME` modules path:
+    - **AllUsers** modules path is inserted before `$PSHOME` modules path
+  - else:
+    - Just use `PSModulePath` as defined since the user deliberately removed
+      the `$PSHOME` location
+
+The **CurrentUser** module path is prefixed only if User scope
+`$env:PSModulePath` doesn't exist. Otherwise, the User scope
+`$env:PSModulePath` is used as defined.
+
+### PowerShell Core 6 startup
+
+PowerShell Core 6 doesn't use contents of `$env:PSModulePath` if it detects it
+was started from PowerShell. It overwrites it with:
+
+- **CurrentUser** modules path + **AllUsers** modules path + `$PSHOME` modules
+  path + Windows PowerShell `$PSHOME` modules path.
+
+### PowerShell 7 startup
+
+In Windows, for most environment variables, if the User-scoped variable exists, a
+new process uses that value only even if a Machine-scoped variable of the same
+name exists.
+
+In PowerShell 7, `PSModulePath` behaves like the `Path` environment variable
+works on Windows. `Path` on Windows is treated differently from other
+environment variables. When a process is started, Windows combines the
+User-scoped `Path` with the Machine-scoped `Path`.
+
+- Retrieve the User-scoped `PSModulePath`
+- Compare to process inherited `PSModulePath` environment variable
+  - If the same:
+    - Append the **AllUsers** `PSModulePath` to the end following the semantics
+      of the `Path` environment variable
+    - The Windows `System32` path comes from the machine defined `PSModulePath`
+      so does not need to be added explicitly
+  - If different, treat as though user explicitly modified it and don't append
+    **AllUsers** `PSModulePath`
+- Prefix with PS7 User, System, and `$PSHOME` paths in that order
+  - If `powershell.config.json` contains a user scoped `PSModulePath`, use that
+    instead of the default for the user
+  - If `powershell.config.json` contains a system scoped `PSModulePath`, use
+    that instead of the default for the system
+
+Unix systems don't have a separation of User and System environment variables.
+`PSModulePath` is inherited and the PS7-specific paths are prefixed if not
+already defined.
+
+### Starting Windows PowerShell from PowerShell 7
+
+For this discussion, _Windows PowerShell_ means both `powershell.exe` and
+`powershell_ise.exe`.
+
+Copy process `$env:PSModulePath` to `WinPSModulePath`:
+
+- Remove PS7 the User module path
+- Remove PS7 the System module path
+- Remove PS7 the `$PSHOME` module path
+
+Use `WinPSModulePath` when starting Windows PowerShell.
```

Comment:
Might be worth noting the reason PS7 paths are removed is so that WinPS doesn't try to load a PS7 module which won't work

Reply:
Fixed